package.json

```diff
@@ -19,14 +19,18 @@
     "grunt-contrib-cssmin": "0.7.0",
     "grunt-contrib-watch": "0.5.3",
     "grunt-sass": "1.1.0",
-    "grunt-swig-templates": "0.1.4",
     "grunt-version": "0.3.3",
     "jit-grunt": "0.9.1",
     "load-grunt-tasks": "0.2.0",
-    "node-bourbon": "4.2.3",
     "grunt-jscs": "2.8.0",
     "grunt-contrib-jshint": "1.0.0"
   },
+  "dependencies": {
+    "grunt-swig-templates": "0.1.4",
+    "node-bourbon": "4.2.3",
+    "incuna-sass": "github:incuna/incuna-sass#6.1.0",
+    "incuna-transitions": "github:incuna/incuna-transitions#1.6.0"
```

Comment:
I think you need to fix the bourbon include paths in Grunt: https://github.com/incuna/style-guide/blob/137a7c092703292ec93713088e10cdce650b5a6a/Gruntfile.js#L40-L41

```js
includePaths: [
    require('node-bourbon').includePaths,
    'bower_components/incuna-sass',
    'bower_components/incuna-transitions',
    'sass'
]
```

Will this work?
```js
includePaths: [
    require('node-bourbon').includePaths,
    require('incuna-sass').includePaths,
    require('incuna-transitions').includePaths,
    'sass'
]
```